### DISCUSSION TITLE
How to add arguments to jq for localb.in

### DISCUSSION THREAD

**Discussion by cdxker** [esin/esin#29] [Guestbook]
Hey really love localb.in. I'm trying to use localb.in/jq, but can't figure out how to add filters like you would for a normal jq impl. 

Also can you follow me?

**Comment by esin** [comment]
hey!

you can try something like this:
`curl https://api.github.com/users/esin -s | curl -d @- 'https://localb.in/jq?.login'` is equivalent for `curl https://api.github.com/users/esin | jq .login`
